openpype/hosts/aftereffects/api/ws_stub.py

```diff
@@ -298,13 +298,14 @@ def import_file(self, path, item_name, import_options=None):
             self.client.call('AfterEffects.import_file',
                              path=path,
                              item_name=item_name,
-                             import_options=import_options)
+                             import_options=import_options,
+                             fps=fps)
             )
         records = self._to_records(self._handle_return(res))
         if records:
             return records.pop()
 
-    def import_file_with_dialog(self, path, item_name):
+    def import_file_with_dialog(self, path, item_name, fps=None):
```

Comment:
fps should be added in the `import_options`, not as a new argument of the function